uninstall/3.1/README.md

```diff
@@ -0,0 +1,37 @@
+# Uninstall Cloud Pak for Watson AIOps
+
+The scripts provided here can be used to uninstall Cloud Pak of Watson AIOps and delete resources created by it.
+
+## Prereqs
+- You need to have OC CLI installed
+- You have logged into your cluster using `oc login`
+
+## Getting started
+
+Clone this repo.
+```
+  git clone https://github.ibm.com/katamari/katamari-util.git 
```

Comment:
@kvijai82 : we should be changing this to current repository and not the katamari-util, right ?

Reply:
Good catch. Updated it.